### ISSUE TITLE
Disabling a row

### ISSUE THREAD

**Issue by rosepriya-jk** [Saulis/iron-data-table#61]
Is there any config to disable a particular row in the table? Or if some config is added for the same, will that be useful?

**Comment by Saulis** [comment]
Hi!

What do you mean by disabling? Preventing selection/expanding when clicking? There isn't any configuration for that, but if you want to achieve that, you can manually add a `tap` event listener for each `data-table-row` elements and call `e.stopPropagation()` in the listener to prevent selection.

**Comment by rosepriya-jk** [comment]
Yeah, selection/expansion should be prevented and the disabled row should have some custom css like a change in text color and background color.Just like the [selected] config. Is it ok if I add that as a feature in grid?

**Comment by Saulis** [comment]
What kind of API were you thinking of? How would the user define which rows to disable?

**Comment by rosepriya-jk** [comment]
thought of having a 'disabled' config just like selected and expanded

**Comment by Saulis** [comment]
OK, the values of  `selected` and `expanded` get toggled between `true` / `false` on user click actions, 
... so how the `disabled` would be set to `true` ?

**Comment by Saulis** [comment]
How about if `<data-table-row disabled>` would prevent any selection and row expanding from happening, and the user would then do:

``` js
  table.beforeRowBind = function(data, row) {
    row.disabled = data.foo === 'bar';
  }
```

**Comment by Saulis** [comment]
@tinkerD any thoughts on this?
